.gitlab-ci.yml

```diff
@@ -210,9 +209,11 @@ variables:
   CLANG_LLVM_VER: 12.0.1
   KERNEL_MATRIX_TESTING_X86_AMI_ID: "ami-0c54d42f8f4180b0c"
   KERNEL_MATRIX_TESTING_ARM_AMI_ID: "ami-021f04c00ecfa8590"
-  RUN_E2E_TESTS: "auto" # Should be "off", "auto" or "on" it will change the trigger condition for new-e2e tests on branch != main
+  RUN_E2E_TESTS: "auto" # Should be "auto", "on", "off" it will change the trigger condition for new-e2e tests on branch != main
+  RUN_UNIT_TESTS: "auto" # Should be "auto", "on", "off" it will change the trigger condition for unit tests on branch != main
   # skip known flaky tests by default
   GO_TEST_SKIP_FLAKE: "true"
+  FAST_TESTS: "true"
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
```suggestion
  FAST_TESTS: "true" # run only tests covering code impacted by changes. `RUN_UNIT_TESTS="on"` override this to `false`
```

Reply:
Wonder if we could keep one single variable using `RUN_UNIT_TESTS`. Looks like `RUN_UNIT_TESTS=="auto"` has the same meaning.

Reply:
I think this is a bit misleading, but this variable is not supposed to be edit by the user triggering the pipeline. It was more to give it a default value. I will make a small change